libs/scripts/delayed.js

```diff
@@ -28,7 +28,9 @@ export const loadPrivacy = async (getConfig, loadScript) => {
   const acom = '7a5eb705-95ed-4cc4-a11d-0cc5760e93db';
   const ids = {
     'hlx.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271-test',
+    'aem.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271-test',
```

Comment:
We will need help from @salonijain3, @bandana147 to get new IDs for Privacy, the `aem.page|live` need allowlisting.